monai/bundle/utils.py

```diff
@@ -108,16 +108,27 @@
         # MLFlowHandler config for the trainer
         "trainer": {
             "_target_": "MLFlowHandler",
+            "_disabled_": "$torch.distributed.get_rank() > 0 if torch.distributed.is_initialized() else False",
```

Comment:
Please also check "dist.is_available()", refer to:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/data/dataset.py#L808
More details:
https://pytorch.org/docs/stable/distributed.html#torch.distributed.is_available

Thanks.

Reply:
Thanks Nic, I will add this check in the initial config.